scripts/plot_performance_over_time.py

```diff
@@ -28,40 +28,43 @@ def lighten_color(input_color, amount=0.5):
     """
     try:
         c = colors.cnames[input_color]
-    except:
+    except Exception:
```

Comment:
What's the reasoning here? (Also in other files)

Reply:
The linter does not like `except` without specifying any Exception type, but yes, `Exception` is not much better. I think this particular case could be solved without exceptions. The other ones are more painful.